Wireless.py

```diff
@@ -195,28 +195,48 @@ def _computeConcurrentTransmissionPDR(self, sender, receiver):
                 relays += [device]
 
         if relays:
+            # find PDR though CT computations
+            pdr = self._getPdrCT(relays, movingNode)
```

Comment:
I don't understand why you have both `_computeConcurrentTransmissionPDR` and `_getPdrCT`